client/state/stored-cards/reducer.js

```diff
@@ -79,6 +88,32 @@ export const isFetching = ( state = false, action ) => {
 	return state;
 };
 
+/**
+ * `Reducer` function which handles request/response actions
+ * concerning stored card editing
+ *
+ * @param {object} state - current state
+ * @param {object} action - storedCard action
+ * @returns {object} updated state
+ */
+export const isEditing = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case STORED_CARDS_EDIT:
+			return {
+				...state,
+				[ action.card.stored_details_id ]: true,
+			};
+
+		case STORED_CARDS_EDIT_FAILED:
+		case STORED_CARDS_EDIT_COMPLETED:
```

Comment:
When a `case` statement creates variables, they will be available outside the `case` as well, which can sometimes pollute other statements. It's best to create a scope for this block by wrapping it in curly braces.

```
case 'something': {
  const myVar = 5;
  return myVar;
}
```

Reply:
This was removed altogether in lieu of calling the API endpoint directly instead of running through the reducer.